providers/openmetrics/options.go

```diff
@@ -0,0 +1,49 @@
+package metrics
+
+import (
+	prom "github.com/prometheus/client_golang/prometheus"
+	"google.golang.org/grpc/status"
+)
+
+// Helper methods
+func FromError(err error) (s *status.Status, ok bool) {
+	return status.FromError(err)
+
+	// TODO: @yashrsharma44 - discuss if we require more error handling from the previous package
```

Comment:
In the grpc prometheus, they had more complex handling of error [here](https://github.com/grpc-ecosystem/go-grpc-prometheus/blob/9abf3eb82b4a313b1a28f370a7ef8ffe0667c681/packages/grpcstatus/grpcstatus.go#L37). Do we want to have this logic here?